jaxrs-spec/src/main/asciidoc/chapters/introduction/_project_team.adoc

```diff
@@ -17,8 +17,6 @@ of the project team and various contributors.
 
 The following are the current committers of the Jakarta RESTful Web Services specification project:
 
--- Sebastian Daschner (IBM)
```

Comment:
I'm wondering why you removed Sebastian as he is still listed here: https://projects.eclipse.org/projects/ee4j.jaxrs/who

Reply:
Sebastian was inactive for very long time and has not contributed to 3.1. That listing on the EF page seems to be rather politically motivated, so the EF never throws out a committer there. I do not see the value of still having him mentioned in the *current* spec document. If I missed some of his merits on the 3.1 time frame, please point me to his actual contributions (issues, review, etc.). I would be happy to keep his name in the spec if he *actually* contributed to 3.1.

Reply:
@mkarg The decision to remove someone should be discussed. Once again, you are introducing changes in a PR that are _not related_ to the PR's title or intent. I'm fine with discussing this but in a separate PR.

Reply:
Then intention of this PR *is* to discuss it. What is the benefit of discussing this single commit in *different* PR? The arguments will be the same.

Reply:
It isn't based on the title or the intent of this PR. If the scope of an update is different from the title/intent, it should be a different PR/discussion. Please create another PR that isn't related to the _technical_ intent of the PR.

Reply:
While I do not see your argument actually (the text in the subject of this PR does not say this is just about "technical" changes; moreover it actually says this is just a bunch of changes) we can separate the PRs if you have such strong feelings about it.

Reply:
Undone committer removal, opened separate PR for this. Please withdraw veto. Thanks.